packages/mosaic/popover/popover.component.ts

```diff
@@ -440,6 +455,10 @@ export class McPopover implements OnInit, OnDestroy {
 
     private manualListeners = new Map<string, EventListenerOrEventListenerObject>();
     private readonly destroyed = new Subject<void>();
+    // @ts-ignore
+    private readonly resizeObserver = new ResizeObserver(() => {
```

Comment:
I do not see any possibility to watch\observe popover size dynamic changes (content size changed) without monitoring it via ResizeObserver, otherwise I do not get an idea on how we should determine if the `window:resize` event should be fired on our side manually.

Reply:
> I do not see any possibility to watch\observe popover size dynamic changes (content size changed) without monitoring it via ResizeObserver.

For most cases, it will be sufficient to use the window resize event.
In other cases, user must trigger window resize event manually.

For example you can see https://github.com/angular/components/blob/e7c7dce13a5a77ca66a92650b79199da697d0734/src/material-experimental/mdc-slider/slider.ts#L262